just_audio_background/lib/just_audio_background.dart

```diff
@@ -443,20 +436,18 @@ class _PlayerAudioHandler extends BaseAudioHandler
     return LoadResponse(duration: response.duration);
   }
 
-  Future<SetVolumeResponse> customSetVolume(SetVolumeRequest request) async {
-    return await (await _player).setVolume(request);
-  }
+  Future<SetVolumeResponse> customSetVolume(SetVolumeRequest request) async =>
```

Comment:
This async can also be deleted.

Reply:
It's not, because of `await player`?

Reply:
Or you want to use `_player.then((p) => p.setVolume())`, that would be a sync call that returns Future?

Reply:
Sorry I missed that one. We need the await and therefore the async.

Reply:
Got it, then I resolve these 2 comments.